lib/command/run-workers.js

```diff
@@ -46,6 +46,12 @@ module.exports = function (workers, options) {
     ...overrideConfigs,
   };
 
+  const configRerun = config.rerun || {};
+  let maxReruns = configRerun.maxReruns || 1;
+  if (maxReruns > 1) {
+    maxReruns = maxReruns * numberOfWorkers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment can be replaced with operator assignment.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)